.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 build
 bridge.log
 package-lock.json
+yarn.lock
```

Comment:
The lock file should be checked in: https://classic.yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control

Reply:
`npm` has the same sort of lock file (`package-lock.json`), but we've always left it out. It causes issues with merging/rebasing, and it pollutes diffs with tons of unimportant changes. See [here](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5#comment78284962_44210813) for similar complaints. Many [major](https://github.com/jquery/jquery) [projects](https://github.com/serverless/serverless) leave it out.

We've never had any serious problems leaving it out. The only issues were those caused by us choosing not to lock down versions in the `package.json` (which `package-lock.json` kinda forces you to do anyway). If your package only works with certain versions, that can be specified in the `package.json` by locking down the package. We should be specifying those versions explicitly if we worry that certain versions break our code.

There are some other technical complaints about how you're locking in versions in a pretty arbitrary way that package maintainers didn't intend: https://dev.to/gajus/stop-using-package-lock-json-or-yarn-lock-3ddi.

Happy to discuss this more, but I think we should leave this issue for a separate thread/PR since our repo currently operates without `package-lock.json`.